cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1743,21 +1783,22 @@ display.page.search.tab = statistics
+search = `cs_gws` sourcetype="gws:reports:drive" "event.type"=acl_change visibility IN ("shared_externally","people_with_link") _index_earliest=-61m@m _index_latest=-1m@m \
 | rex field=owner "[^@]+@(?<src_domain>[^@]+)" \
 | rex field=email "[^@]+@(?<dest_domain>[^@]+)" \
+| where isnotnull(src_domain) AND isnotnull(dest_domain) \
 | search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain` \
-| stats values(doc_title) as doc_title, values(doc_type) as doc_type, values(email) as dst_email_list, values(visibility) as visibility, count, min(_time) as firstTime, max(_time) as lastTime by owner, ipAddress \
 | `cs_user_privilege_mapping(owner)` \
+| stats values(doc_title) as doc_title, values(doc_type) as doc_type, values(event.name) as event.name, values(email) as dst_email_list, values(visibility) as visibility, count, min(_time) as firstTime, max(_time) as lastTime by owner \
 | eval cyences_severity = if(is_privileged_user=="Yes", "critical", "medium") \
 | `cs_human_readable_time_format(firstTime)` \
 | `cs_human_readable_time_format(lastTime)` \
 | `cs_gws_drive_object_shared_outside_filter`
 action.cyences_notable_event_action = 1
 action.cyences_notable_event_action.param.filter_macro_name = cs_gws_drive_object_shared_outside_filter
-action.cyences_notable_event_action.contributing_events = `cs_gws` sourcetype="gws:reports:drive" NOT email IN ("", "null") | rex field=owner "[^@]+@(?<src_domain>[^@]+)" | rex field=email "[^@]+@(?<dest_domain>[^@]+)" | search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain` 
-action.cyences_notable_event_action.system_compromised_search = | stats count by owner, ipAddress
-action.cyences_notable_event_action.system_compromised_drilldown = `cs_gws` sourcetype="gws:reports:drive" NOT email IN ("", "null") | rex field=owner "[^@]+@(?<src_domain>[^@]+)" | rex field=email "[^@]+@(?<dest_domain>[^@]+)" | search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain` | search owner=$row.owner$ ipAddress=$row.ipAddress$
+action.cyences_notable_event_action.contributing_events =  `cs_gws` sourcetype="gws:reports:drive" "event.type"=acl_change visibility IN ("shared_externally","people_with_link") _index_earliest=-61m@m _index_latest=-1m@m | rex field=owner "[^@]+@(?<src_domain>[^@]+)" | rex field=email "[^@]+@(?<dest_domain>[^@]+)" | where isnotnull(src_domain) AND isnotnull(dest_domain) 
+action.cyences_notable_event_action.system_compromised_search = | stats count by owner
+action.cyences_notable_event_action.system_compromised_drilldown =   `cs_gws` sourcetype="gws:reports:drive" "event.type"=acl_change visibility IN ("shared_externally","people_with_link") _index_earliest=-61m@m _index_latest=-1m@m | rex field=owner "[^@]+@(?<src_domain>[^@]+)" | rex field=email "[^@]+@(?<dest_domain>[^@]+)" | where isnotnull(src_domain) AND isnotnull(dest_domain) |search owner=$rawdata.owner$
```

Comment:
Add filter before drilldown: | search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain`